ENIGMAsystem/SHELL/Universal_System/roomsystem.cpp

```diff
@@ -232,7 +232,7 @@ namespace enigma
     // In pull request 1831, it was decided to adopt GMS behavior instead of GM8.
     // The window is no longer moved, centered, or resized when switching rooms.
     // This is always true, even if the room sizes are different.
-    enigma_user::window_default(true);
+    enigma_user::window_default(false);
```

Comment:
Is the above comment stale, now, or was it stale at HEAD?

Reply:
The comment is stale in master. I accidentally reverted the fact we adopted gmstudio behavior by making the argument true, this sets it back to false.